packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -355,6 +364,24 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   final bool obscureText;
 
+  /// {@template flutter.widgets.editableText.readOnly}
+  /// Whether the text can be changed.
+  ///
+  /// When this is set to true, the text cannot be modified
+  /// by any shortcut or keyboard operation. The text is still selectable.
+  ///
+  /// Defaults to false. Must not be null.
+  /// {@endtemplate}
+  final bool readOnly;
+
+  /// Whether to show selection handles.
```

Comment:
It may be good to explain in a sentence or two what the selection handle is and how it is different from the cursor. Maybe also link to showCursor?